eclcli/vnf/v1/virtual_network_appliance.py

```diff
@@ -121,7 +122,7 @@ def get_parser(self, prog_name):
                    "virtual network appliance. "
                    "net-id: attach interface to network with this UUID, "
                    "ip-address: IPv4 fixed address for interface. "
-                   "(You can specify only one address in creation), "
+                   "(You can specify one or more addresses in creation), "
```

Comment:
```suggestion
                   "You can specif multiple ip address by using ':' "
                   "(e.g: 1.1.1.1:2.2.2.2:...), "
```
UpdateVirtualNetworkApplianceInterfacesのhelp文言と揃えていただいてもよろしいでしょうか

Reply:
helpの修正をありがとうございます。